src/main/java/com/stripe/model/ExpandableFieldDeserializer.java

```diff
@@ -0,0 +1,45 @@
+package com.stripe.model;
+
+import com.google.gson.*;
+import java.lang.reflect.ParameterizedType;
+import java.lang.reflect.Type;
+
+public class ExpandableFieldDeserializer implements JsonDeserializer<ExpandableField> {
+    public ExpandableField deserialize(JsonElement json, Type typeOfT, JsonDeserializationContext context) throws JsonParseException {
+        Gson gson = new GsonBuilder()
+                .setFieldNamingPolicy(FieldNamingPolicy.LOWER_CASE_WITH_UNDERSCORES)
+                .create();
+
+        if (json.isJsonNull()) {
+            return null;
+        }
+
+        ExpandableField expandableField;
+
+        // Check if json is a String ID:
+        if (json.isJsonPrimitive()) {
+            JsonPrimitive jsonPrimitive = json.getAsJsonPrimitive();
+            if (jsonPrimitive.isString()) {
+                expandableField = new ExpandableField(jsonPrimitive.getAsString(), null);
+                return expandableField;
+            }
+            else {
+                throw new JsonParseException("ExpandableField is a non-string primitive type.");
+            }
+        }
+        // Check if json is an expanded Object:
```

Comment:
And similarly here. e.g. "Check if the JSON data is an object. If it is, it's an indication that the field has been expanded in the response."